bin/mail/2016-08-09-hoc-latam/generate-mailing-lists.rb

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# Latin America (latam), 2 sub-lists for es & pt
+
+LATAM_ES_COUNTRIES = [
+  'Mexico',
+  'Guatemala',
+  'El Salvador',
+  'Costa Rica',
+  'Nicaragua',
+  'Belize',
+  'Panama',
+  'Cuba',
+  'Dominican Republic',
+  'Venezuela',
+  'Colombia',
+  'Ecuador',
+  'Peru',
+  'Bolivia',
+  'Paraguay',
+  'Uruguay',
+  'Chile',
+  'Argentina',
+  'Puerto Rico'
+]
```

Comment:
super nit: Use `[...].freeze` to assure the SCREAMING_CASE_CONSTANT remains constant.
(eventually this rubocop will get enforced)